lib/rucio/web/rest/flaskapi/v1/archive.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+'''
+  Copyright European Organization for Nuclear Research (CERN)
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+  Authors:
+  - Vincent Garonne, <vincent.garonne@cern.ch>, 2017
+  - Thomas Beermann, <thomas.beermann@cern.ch>, 2018
+'''
+
+from json import dumps
+from logging import getLogger, StreamHandler, DEBUG
+from traceback import format_exc
+
+from flask import Flask, Blueprint, Response
+from flask.views import MethodView
+
+from rucio.api.did import list_archive_content
+from rucio.web.rest.flaskapi.v1.common import before_request, after_request
+
+LOGGER, SH = getLogger("rucio.meta"), StreamHandler()
```

Comment:
rucio.meta ? shouldn't this be name of the module?

Reply:
Yes, you're probably right. tbh I just copied the file from webpy, changed it to work with Flask and added the documentation. So, if there are any bugs or else I wouldn't have actually fixed them.